src/genericComponents/Button.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import styled from 'styled-components';
+
+const ButtonPrimary = (props: any) => {
+  const { onClick, text, variant } = props;
+
+  // className={`${getVariantClassName(variant)}${className ? ' ' + className : ''}`}
+
+  return (
+    <CAGButton
+      className={`CAGButton.${variant}-class`}
+      onClick={onClick}
+      variant={variant}
+    >
+      {text}
+    </CAGButton>
+  );
+};
+
+const CAGButton = styled(Button)`
+  &.primary-class {
+    width: 96px;
```

Comment:
We don't want to copy and paste all of the CSS directly from Adobe xd. Some notes for these properties:
1. Do not give an exact width; button width will change based on length of text
2. Height should be derived from equal top and bottom padding + text height and line height. Giving an absolute height risks the text not being vertically centered
3. For setting fonts and colors, we want to only use variables we define in our theme/styleVars.ts file. So you want to import fonts and colors from there and use their properties here in the style block
4. Since you're not providing a background image, just a color, you _only_ need the color. The other options Adobe Xd throws in there are unnecessary. 
4. Instead of having an alpha channel on your black hex for box-shadow, you should use rgba - it's more widely supported
5. Opacity is not needed
6. For the things all of your variant classes share, you can put them above the classes as part of the CAGButton's generic styles. No need to have redundancy in that regard
7. Make sure all of your style properties are listed alphabetically